pkg/config/schema.go

```diff
@@ -155,27 +167,63 @@ func (check *SchemaCheck) Initialize(id string) error {
 		}
 		check.SchemaString = string(jsonBytes)
 	}
-	err := json.Unmarshal([]byte(check.SchemaString), &check.Validator)
-	return err
+	for kind, schema := range check.AdditionalSchemas {
+		jsonBytes, err := json.Marshal(schema)
+		if err != nil {
+			return err
+		}
+		check.AdditionalSchemaStrings[kind] = string(jsonBytes)
```

Comment:
Since AdditionalSchemas and ADditionalSchemaStrings can both be specified we could be overwriting a schema string here.

Reply:
Documented